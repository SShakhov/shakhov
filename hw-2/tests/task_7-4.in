1My2Str3ING wIth S4paces
