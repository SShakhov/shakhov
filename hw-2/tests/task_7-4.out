1My2Str3ING wIth S4paces
1. All numbers in front in the same order
1234MyStrING wIth Spaces
